tasks.py

```diff
@@ -153,6 +153,15 @@ def tests(c):
         docker_compose_run(c, 'php ./vendor/bin/simple-phpunit')
 
 
+@task
+def phpstan(c):
+    """
+    Runs PHPStan
+    """
+    with Builder(c):
+        docker_compose_run(c, 'php ./vendor/bin/phpstan analyse')
```

Comment:
It should be:

```suggestion
        docker_compose_run(c, 'vendor/bin/phpstan analyse')
```

But I see all the file is like that :( so let's keep it as it